MTFHDataLanding/UseCase/LandPersonData.cs

```diff
@@ -19,63 +19,40 @@ namespace MTFHDataLanding.UseCase
     public class LandPersonData : ILandPersonData
     {
         private readonly IPersonApi _personApi;
-        private readonly ILogger<LandPersonData> _logger;
         private readonly IAmazonS3 _s3Client;
 
         public LandPersonData(IPersonApi personApi,
-            ILogger<LandPersonData> logger,
             IAmazonS3 s3Client)
         {
             _personApi = personApi;
-            _logger = logger;
             _s3Client = s3Client;
         }
 
         [LogCall]
         public async Task ProcessMessageAsync(EntityEventSns message)
         {
-            var fileTransferUtility = new TransferUtility(_s3Client);
-
             if (message is null) throw new ArgumentNullException(nameof(message));
 
             // 1. Get Person from Person service API
             var person = await _personApi.GetPersonByIdAsync(message.EntityId, message.CorrelationId)
-                                         .ConfigureAwait(false);
+                .ConfigureAwait(false);
             if (person is null) throw new PersonNotFoundException(message.EntityId);
 
-            using (var jsonReader = new ChoJSONReader(new JsonTextReader(new StringReader(JsonConvert.SerializeObject(person)))))
-            {
-                using (var memoryStream = new MemoryStream())
-                {
-                    using (var w = new ChoParquetWriter(memoryStream))
-                    {
-                        w.Write(jsonReader);
-                        _logger.LogWarning($"Person record (id: {person.Id}): ");
-
-                        //await fileTransferUtility.UploadAsync(memoryStream, Constants.BUCKET_NAME, Constants.KEY_NAME + "year=" + year + "/month=" + month + "/day=" + day + "/" +
-                        //    message.DateTime.ToString("HH\\:mm\\:ss.fffffff") + ".parquet");
-                    }
 
-                    string year = message.DateTime.ToString("yyyy");
-                    string month = message.DateTime.ToString("MM");
-                    string day = message.DateTime.ToString("dd");
+            using (var memoryStream = new MemoryStream(Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(person))))
```

Comment:
How many records are in each file?

Reply:
From what I understand - 1 record/file

Reply:
We'll need to batch records, otherwise we'll end up with 10s of thousands of single record files. Any application will grind  to a halt when trying to read them in. What is the volume expected with this application?

Reply:
Not sure how this is possible with a Publish/queue/listener system. You could read them sequentially at a specific moment in the day? (As for the volume question I cannot say)

Reply:
I hear your point, however it is impractical to have a single record per file. A means of getting around this would be to append to existing records, however this will result in additional complexity and memory requirements in the lambda.

If SQS has these constraints would it be possible to consider alternative tooling such as kafka / kinesis? Tools such as kafka / kinesis have connectors for tools such as spark streaming that would allow us to micro batch records from a stream which would resolve this issue. These tools have primitives that allow us to create a windows for processing.

Reply:
Hey, it's not so much how the data is read from our end, more that dropping files like this is a very bad practice and will  make it harder to process and manage going forward. Lots of little files will make jobs more brittle and will restrict the volume we can process affecting time to insight. 

Using a tool such as kafka or kinesis would reduce the amount of code on your end and make things simpler long term for all. It's not a silver bullet, and there are other things we do need to address when streaming data into the data environment that have not been covered, but I'm just thinking about us not reinventing the wheel to solve certain problems.